cl/sentinel/handlers/light_client.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/libp2p/go-libp2p/core/network"
 )
 
+const maxLightClientsPerRequest = 100
```

Comment:
@Giulio2002 How about changing the value of `c.beaconConfig.MaxRequestLightClientUpdates` from config? 

https://github.com/ledgerwatch/erigon/blob/d6a10a4aedf772f49af8789aeaa4a7c8f8f2ef0e/cl/clparams/config.go#L809

Reply:
i dont think that should actually be kept in beacon config. i think this is fine. i removed that field btw from the config

Reply:
Okay. Then it is good.